### ISSUE TITLE
File cut

### ISSUE THREAD

**Issue by prettydiff** [prettydiff/shared-spaces#17]
This is already working when both the origination and destination are localhost.

This must be enabled for:

* local to remote
* remote to local
* remote to remote

In the case of *local to remote* or *remote to local* file deletion must occur as a separate task after the files are written and hashes compared.

**Blocked by #16**, file deletion does not appear to work on remote agents.

**Comment by prettydiff** [comment]
- [x] This is already working when moving from one location to another on the same computer whether remote or localhost.
- [ ] Test when cutting a document from localhost to remote
- [ ] Test when cutting a document from remote to localhost
- [x] Opening an issue for cutting from one remote to a different remote #24

**Comment by prettydiff** [comment]
Blocked by #18

**Comment by prettydiff** [comment]
Issues #15 and #18 are largely complete and this issue is being worked in branch `writteThrottling`.  It appears to be working at this time, but there is a regression:

Provided the browser displays two file navigator modals, one the localhost and one for the remote computer.  If these two modals point to the same file system address the cut action updates the wrong modal to show file removal.  The process is working correctly on the file systems, but communicates incorrect information to the user.

**Comment by prettydiff** [comment]
The prior described problem occurs in the function `terminal_server_fileService_watch_remote` the result of file system watch from the remote,

The problem has 2 different results:

1. **cut from localhost to remote** - The remote modal is updated as though files were removed, but the files were removed from localhost
2. **cut from remote to localhost** - The remote modal is not updated to indicate any files were removed, but they were

The locations were the files are written update correctly whether localhost or remote.
